src/extension.ts

```
@@ -225,6 +232,15 @@ export async function activate(context: vscode.ExtensionContext) {
   return {
     loadLanguage,
 
+    registerLanguage(languageId, wasm) {
+      if (languages[languageId] != null) {
+          return;
```

I think I'd throw an exception in this case